src/types/algoliasearch.ts

```diff
@@ -25,6 +26,8 @@ import {
   // eslint-disable-next-line import/no-unresolved
 } from '@algolia/client-search';
 
+export { FindAnswersResponse };
```

Comment:
is this `any` on v3 or an error?

Reply:
I created an empty repo with v3 to see how it goes, but VSCode didn't tell me anything.
To make it explicit: https://github.com/algolia/instantsearch.js/pull/4581/commits/f0ccf1c775c01d0886e9f4e199b80ebb2bdf68d6